EventPlanning.BL/EventManager.cs

```diff
@@ -155,6 +155,29 @@ public Event CopyEventDataToEvent(EventData eventData)
             return newEvent;
         }
 
+        public bool PatchEvent(EventData eventData, int eventId)
+        {
+            var ev = repository.GetEvent(eventId);
+
+            if (ev == null)
+            {
+                return false;
+            }
+
+            ev.Address = eventData.Address != ev.Address ? eventData.Address : ev.Address;
+            ev.DateFrom = Convert.ToDateTime(eventData.DateFrom) != (ev.DateFrom) ? Convert.ToDateTime(eventData.DateFrom) : ev.DateFrom;
```

Comment:
The `Convert.ToDateTime` is twice here if it's not equal, let's move that out (reduce computing).
Also `(ev.DateFrom)` parentheses are redundant